lib/mshoplib/src/MShop/Order/Item/Base/Default.php

```diff
@@ -665,6 +665,33 @@ public function getEditor()
 
 
 	/**
+	 * Sets the item values from the given array.
+	 *
+	 * @param array $list Associative list of item keys and their values
+	 * @return array Associative list of keys and their values that are unknown
+	 */
+	public function fromArray( array $list )
+	{
+		$unknown = array();
+
+		foreach( $list as $key => $value )
+		{
+			switch( $key )
+			{
+				case 'order.base.id': $this->setId( $value ); break;
+				case 'order.base.comment': $this->setComment( $value ); break;
+				case 'order.base.customerid': $this->setCustomerId( $value ); break;
+				case 'order.base.status': $this->setStatus( $value ); break;
+				case 'order.base.languageid': $this->_locale->setLanguageId( $value ); break;
+				default: $unknown[$key] = $value;
+			}
+		}
+
+		return $unknown;
```

Comment:
This implementation is different from the others due to the missing inheritance. What about the ctime, mtime and editor properties in this case?